ValueError('Invalid limits given: (((y, 10, 0), (y, 6, 5)),)',)
